index.html

```diff
@@ -20,27 +20,40 @@
     <link rel="stylesheet" href="./css/styles.css" />
     <title>Portfolio website</title>
   </head>
-  <body>
+  <body class="body">
     <!-- header section  -->
     <header class="header">
       <a href="#" class="logo animate-logo">John Kibirige</a>
       <nav class="header-links">
-        <a href="#portfolio">Portfolio</a>
-        <a href="#about">About</a>
-        <a href="#contact">contact</a>
+        <ul class="nav-menu">
+          <li>
+            <a href="#portfolio">Portfolio</a>
+            <img src="./images/chevron.svg" alt="chevron" />
+          </li>
+          <li>
+            <a href="#about">About</a
+            ><img src="./images/chevron.svg" alt="chevron" />
+          </li>
+          <li>
+            <a href="#contact">contact</a
+            ><img src="./images/chevron.svg" alt="chevron" />
+          </li>
+        </ul>
```

Comment:
- [ ] Good job on your menu work :+1: but there are some issues you need to address :cry: Please try to adjust your mobile menu `background-image` so can you can see all the menu options and options right arrow very well. Right now it is a little bit different from the Figma template. :+1: 

- [ ] I would like to recommend you work on **the close (X) button**, Please try to adjust the size and use the same color  in order to make it similar to the Figma template :+1:  


### Your project
![Screenshot from 2022-07-19 09-47-08](https://user-images.githubusercontent.com/6615394/179726094-13827b57-a5d8-4224-9854-f7a167d5afed.png)

### Figma template
![Screenshot from 2022-07-19 09-46-13](https://user-images.githubusercontent.com/6615394/179726096-2546fd32-bab5-4e12-9f7c-860c55feca79.png)

- [ ] Well done, when I click on a menu option, it leads me to the part mentioned (example: **About/Portfolio/contact**) :100: But it's impossible for me to scroll up or down afterward. I have to refresh the page, please fix this to give the user a pleasant user experience. :smile: 

https://www.loom.com/share/ed8f08191c74406e89930a393a3749e0